### PR TITLE
Migrate git/printable.go to the new world

### PR DESCRIPTION
**Pull request by kevgo**

### CODE REVIEW

File: src/cli/printable.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import (
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/git-town/git-town/src/util"
+)
+
+// PrintableBranchAncestry provides the branch ancestry in CLI printable format.
+func PrintableBranchAncestry(config Config) string {
```
Comment (kevgo) [inline src/cli/printable.go:12]: These functions format configuration data for printing on the CLI. They could go either into the `cli` package (as is done here) or into the `cmd` package. I tried both versions, the `cli` package feels a bit better. But I'm open to suggestions here.

Comment (charlierudolph) [reply src/cli/printable.go:12]: Here seems good.

File: src/cli/config.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package cli
+
+// Config defines the configuration values needed by the `cli` package.
+type Config interface {
+	GetBranchAncestryRoots() []string
+	GetChildBranches(string) []string
+}
```
Comment (charlierudolph) [inline src/cli/config.go:7]: This interface is pretty specific to input for `PrintableBranchAncestry`. Thoughts on moving it into the printable file and calling it `BranchAncestryConfig`?

File: src/cli/printable.go (hunk 1 above)
Comment (kevgo) [reply src/cli/printable.go:12]: Thanks!

File: src/cli/config.go (hunk 2 above)
Comment (kevgo) [reply src/cli/config.go:7]: I'm all for keeping interfaces small and focused. #1546 implements this suggestion.

**Pull request merged**